tests/perf-regression.100threads.30M-keys.yaml

```diff
@@ -10,9 +10,8 @@ user_prefix: 'perf-regression-VERSION'
 failure_post_behavior: destroy
 space_node_threshold: 644245094
 ami_id_db_scylla_desc: 'VERSION_DESC'
-ip_ssh_connections: 'private'
+#ip_ssh_connections: 'private'
```

Comment:
Why do you comment this?  I saw you still use private ssh connection in the new file perf-regression-mv.100threads.30M-keys-i3.yaml

Reply:
It's original version of this yaml file. I didn't change it